docs/release-notes/2.32.0.md

```diff
@@ -159,3 +159,23 @@ If the value of the parameter `max-query-length` is set to 3, 2 or 1, the PMM ag
 The value of the parameter should be greater than 3.
 
 
+[PMM-11126](https://jira.percona.com/browse/PMM-11126): **PMM server cannot be upgraded to version 2.32.0**
+
+A bug in PMM Server ansible scripts caused PMM to upgrade Nginx's dependencies without updating Nginx itself. Due to this, PMM throws an error while upgrading and cannot upgrade to a newer version.
+
+
+!!! caution alert alert-warning "Important"
+    This issue persists on versions prior to PMM 2.32.0.
+
+**Solution**
```

Comment:
@atymchuk should we also restart update somehow. I remember @Nailya suggested to do that in command line as well?

Reply:
Not until we publish 2.33.0. The update should work again from there on.

Reply:
yeah, but this are RNs for 2.33, so it is exactly the time when it will be published :)

Reply:
no, we don't need to restart update, if update wasn't started yet. And even if user changes the line during the upgrade, then it will work. But if update was started and failed, then use should change the file and then start update again with pmm-update -perform command